faqs.md

```diff
@@ -17,12 +17,12 @@ frequently pop up.
 * [Why does `require 'some-gem'` fail?](#why-does-require-some-gem-fail)
 * [Why does require return false when loading a file from a gem?](#why-does-require-return-false-when-loading-a-file-from-a-gem)
 
-We also answer questions on the [RubyGems Support](http://help.rubygems.org/) site and on IRC
-in #rubygems. Some of the information you can find on the support site includes:
+We also answer questions on IRC in #rubygems.
```

Comment:
We should update this to https://bundler.slack.com/

Reply:
I did not know that. Can we use https://slack.bundler.io/ instead since it is helpful for new community members as well as existing ones.